pymc_marketing/mmm/components/base.py

```diff
@@ -556,3 +573,26 @@ def apply(self, x: pt.TensorLike, dims: Dims | None = None) -> TensorVariable:
         """
         kwargs = self._create_distributions(dims=dims)
         return self.function(x, **kwargs)
+
+
+def _serialize_value(value: Any) -> Any:
```

Comment:
Can you please add a test for this function :) ?

Reply:
Done. All serializations are tested (through various tests.)